test/TokenCuratedRegistry.behavior.js

```diff
@@ -55,26 +55,9 @@ function shouldBehaveLikeTokenCuratedRegistry ({
           await this.registry.setChallenge(itemId, this.challenge.address)
         })
 
-        describe('and the challenge has failed', function () {
-          beforeEach(async function () {
-            await this.challenge.set_mock_passed(false)
-            await this.registry.remove(itemId)
-          })
-
-          it('removes the item', async function () {
```

Comment:
Challenge should be resolved, or else ItemOwner will never get reward, and tokens will be locked in the registry

Reply:
nice find! it simplifies this a lot, that was totally an unnecessary require check